packages/sysutils/systemd/config/system.d/cifs.mount.sample

```diff
@@ -15,7 +15,7 @@ Requires=network-online.service
 # 'network-online.service' fails we can not mount and this scripts fails too
 After=network-online.service
 
-# usually we mount networks shares because we want they avaible *before* XBMC starts.
+# usually we mount networks shares because we want them available *before* XBMC starts.
```

Comment:
"usually we mount network shares because we want them available *before* Kodi starts."